hawkbit-sdk/hawkbit-sdk-device/pom.xml

```diff
@@ -35,5 +35,11 @@
          <artifactId>hawkbit-ddi-api</artifactId>
          <version>${project.version}</version>
       </dependency>
+
```

Comment:
maybe should not be used for DDI which is simulating device side

Reply:
yes.. this is needed for the Authentication setup (feign client to get the security tokens/gateway tokens).. so that sdk-device has the utilities to setup/integrate device.

We could separate it (this utility SetupHelper) in different module meaning - if one wants to work with the SDK and have that nice functionality to have the 2 dependencies
-module1 sdk-device
-module2 wherever we decide to put the SetupHelper